Multisig/UI/UI Library/Views/TableViewCell/CardTableViewCell/CardTableViewCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  CardTableViewCell.swift
+//  Multisig
+//
+//  Created by Moaaz on 1/21/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class CardTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var bodyLabel: UILabel!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var iconImageView: UIImageView!
+    @IBOutlet weak var containerView: UIView!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+        containerView.layer.cornerRadius = 20.0
```

Comment:
just fyi - we have a card-background image, and you can use just that - take a look at some xibs that are using it

Reply:
Using the image will make it harder to implement dark mode :), that's why I used it like this

Reply:
really? I thought we can have different images for different appearances right in the asset 🤔

Reply:
Yes we can do so, but we need to create another image instead of just having different value for the shadow color